docs/DeveloperGuide.md

```diff
@@ -496,12 +628,97 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given student is invalid/cannot be found.
+* 3a. The inputted student's name is invalid or cannot be found.
 
     * 3a1. EduManage shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 3.
+
+* 3b. The inputted task index is invalid.
+
+    * 3b1. EduManage shows an error message.
+
+      Use case resumes at step 3.
+
+***
+
+**Use case: Update a task for a student**
+
+**MSS**
+
+1.  Tuition teacher requests to view all tasks for all students.
+2.  EduManage shows a list of students and their tasks.
+3.  Tuition teacher requests to update a task from a specific student in the list by inputting the student's name, a task index and either a new task description or a new task deadline.
```

Comment:
Tuition teacher requests to update a task from a specific student's task list by inputting the student's name, a task index and either a new task description or a new task deadline.
Comment